<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskTwoDialog</class>
 <widget class="QDialog" name="TaskTwoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>401</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>197</width>
     <height>220</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="weatherLabel">
      <property name="text">
       <string>Weather:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="weatherLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="usdLabel">
      <property name="text">
       <string>USD:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="usdLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="euroLabel">
      <property name="text">
       <string>EURO:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="euroLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="showInfoPushButton">
      <property name="text">
       <string>Show Information</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>611</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="openPushButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="misc_files.qrc">
        <normaloff>:/misc_files/pictures/folder.png</normaloff>:/misc_files/pictures/folder.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="aboutPushButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="misc_files.qrc">
        <normaloff>:/misc_files/pictures/help.png</normaloff>:/misc_files/pictures/help.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="misc_files.qrc"/>
 </resources>
 <connections/>
</ui>
